sql/cache.go

```diff
@@ -2,19 +2,25 @@ package sql
 
 import (
 	"fmt"
-	"hash/crc64"
 	"runtime"
 
+	"github.com/cespare/xxhash"
+
 	lru "github.com/hashicorp/golang-lru"
 	errors "gopkg.in/src-d/go-errors.v1"
 )
 
-var table = crc64.MakeTable(crc64.ISO)
-
 // CacheKey returns a hash of the given value to be used as key in
 // a cache.
-func CacheKey(v interface{}) uint64 {
-	return crc64.Checksum([]byte(fmt.Sprintf("%#v", v)), table)
+func CacheKey(v Row) (uint64, error) {
```

Comment:
what about just `Hash()` or `HashOf`. there are lots of ways to key into a cache

Reply:
Done